charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml

```diff
@@ -3,17 +3,32 @@ groups:
   rules:
   # Recording rules for pod usage
   - record: seed:container_cpu_usage_seconds_total:sum_by_pod
-    expr: sum(rate(container_cpu_usage_seconds_total[5m])) by (pod, namespace)
+    expr: sum(rate(container_cpu_usage_seconds_total[5m])) by (pod, container, namespace)
 
   - record: seed:container_memory_working_set_bytes:sum_by_pod
-    expr: sum(container_memory_working_set_bytes) by (pod, namespace)
+    expr: sum(container_memory_working_set_bytes) by (pod, container, namespace)
 
   - record: seed:container_network_receive_bytes_total:sum_by_pod
-    expr: sum(rate(container_network_receive_bytes_total[5m])) by (pod, namespace)
+    expr: sum(rate(container_network_receive_bytes_total[5m])) by (pod, container, namespace)
 
   - record: seed:container_network_transmit_bytes_total:sum_by_pod
     expr: sum(rate(container_network_transmit_bytes_total[5m])) by (pod, namespace)
 
+  # Recording rules for pod requests
+  - record: seed:kube_pod_container_resource_requests_cpu_cores:sum_by_pod
+    expr: sum(kube_pod_container_resource_requests_cpu_cores) by (pod, container, namespace)
```

Comment:
This comes from the kube-state is each control place. To get this you need to get the federated metrics from each shoot prometheus on the seed.